### PR TITLE
fix 'Unstable message sandbox test' (close #410)

### LINKED ISSUE #410: Unstable message sandbox test
`sandbox/event/message-test.js`, `timeout` test.

See #404

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @churkin @AlexanderMoskovkin

### CODE REVIEW

**Comment by miherlosev** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by miherlosev** [comment]
ping @churkin

File: test/client/fixtures/sandbox/event/message-test.js [hunk 1]
```diff
@@ -275,13 +275,42 @@ asyncTest('timeout', function () {
...
+    var storedDelay          = messageSandbox.PING_IFRAME_TIMEOUT;
+    var timeout              = 200;
+
+    messageSandbox.PING_IFRAME_TIMEOUT = 5;
+
+    iframe.src = 'http://cross.domain.com/';
+    window.QUnitGlobals.waitForIframe(iframe)
+        .then(function () {
+            return Promise.all([
+                messageSandbox.pingIframe(iframe, 'pingCmd')
```
Comment (churkin) [inline test/client/fixtures/sandbox/event/message-test.js]: We try not to use nested promises

Comment (miherlosev) [inline test/client/fixtures/sandbox/event/message-test.js]: It is not nested Promise.

Nested Promise (pseudo code):
```javascript
Promise
    .then(function (){
        new Promise.then(function (){
              //some code
        })
    });
```

Non nested Promise, chain of promises (my code):
```javascript
Promise
    .then(function () {
         return Promise;        
    })
    .then(function (){})
```

**Comment by miherlosev** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by miherlosev** [comment]
FPR

**Comment by miherlosev** [comment]
ping @churkin @AlexanderMoskovkin

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by miherlosev** [comment]
FPR

**Comment by churkin** [comment]
lgtm

**Pull request merged**